React/Views/RCTRefreshControl.m

```diff
@@ -37,7 +74,13 @@ - (void)setRefreshing:(BOOL)refreshing
 {
   if (super.refreshing != refreshing) {
```

Comment:
I don't think there is a reason why it is super either, I'll change it to self instead.